### PR TITLE
Disable account creation if there's none

### PR DESCRIPTION
**Pull request by gumb0**

### CODE REVIEW

**Comment by chriseth** [comment]
I think "signingKey" can be completely removed.

**Comment by gumb0** [comment]
The only effect I see so far after removing account generation is the output like this:
```
Transaction Signer: XE50000000000000000000000000000000 (00000000-0000-0000-0000-000000000000 - 00000000)
Mining Beneficiary: XE50000000000000000000000000000000 (00000000-0000-0000-0000-000000000000 - 00000000)
Foundation: XE55PXQKKK4B9BYPBGT1XCYW6R5ELFAT6EM (00000000-0000-0000-0000-000000000000 - de0b2956)
```

**Comment by chriseth** [comment]
Yes, I would also remove the line starting with `Foundation` and the one starting with `Transaction Signer` - there should be no default transaction signer.

File: eth/main.cpp [hunk 1]
```diff
@@ -962,10 +959,7 @@ int main(int argc, char** argv)
 	for (auto const& s: passwordsToNote)
 		keyManager.notePassword(s);
 
-	writeFile(configFile, rlpList(signingKey, author));
-
-	if (sessionKey)
-		signingKey = sessionKey;
+	writeFile(configFile, rlpList(author, author));
```
Comment (gumb0) [inline eth/main.cpp]: Not sure about this, probably we can't change the format of this file easily? Or do we need it at all

Comment (gumb0) [inline eth/main.cpp]: Anyway this author is being read in the beginning from this file if written previously

And then passed to eth::Client::setAuthor (which sets it to some "preSeal" block)

Comment (gumb0) [inline eth/main.cpp]: Ok, that's what I did here

**Comment by gumb0** [comment]
Removed --import-session-secret, --sign-key, --session-sign-key options,

**Pull request merged**